### PR TITLE
Release v0.9.0

### PR DESCRIPTION
**Pull request by henriquesimoes**
I've followed the procedure described in [RELEASE.md](https://github.com/cnpem/epics-in-docker/blob/60f793ae94719e3bd18f16dde4d560681763f2f8/RELEASE.md). I would be good to have a second (and third) pair of eyes taking a look before merging.

### CODE REVIEW

**Comment by henriquesimoes** [comment]
@flowln, could you see if my release description is good enough for those who had the same issue as you?

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## v0.9.0
+
+Users willing to use IMPIComm, IOCStats, pyDevSup or SNMP modules should update
```
Comment (flowln) [inline CHANGES.md:7]:
```suggestion
Users willing to use IPMIComm, IOCStats, pyDevSup or SNMP modules should update
```

File: CHANGES.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+to this release. Anyone who had trouble with versions greater than v0.7.0
+building IOCs depending on Calc due to Sequencer linkage issues may find it
```
Comment (flowln) [inline CHANGES.md:9]:
```suggestion
building IOCs depending on Calc due to Sequencer linkage issues should find it
```

File: CHANGES.md (hunk 1 above)
Comment (henriquesimoes) [reply CHANGES.md:7]: Good catch. Thanks!

Comment (gustavosr8) [inline CHANGES.md:7]: The official [iocStats repo](https://github.com/epics-modules/iocStats) writes the name like this. What do you think?
```suggestion
Users willing to use IMPIComm, iocStats, pyDevSup or SNMP modules should update
```

Same in https://github.com/cnpem/epics-in-docker/blob/3d27ac658a1a9062f5314198ee12afafbdec2b4b/CHANGES.md?plain=1#L19 (my bad, sorry)

Comment (henriquesimoes) [reply CHANGES.md:7]: > What do you think?

Let's follow it. Thanks!

**Comment by henriquesimoes** [comment]
I'll rebase this as soon as #64 gets merged, so we create a single release with all pending changes.

**Review by gustavosr8 — APPROVED** [review]

**Pull request merged**